src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -235,6 +235,12 @@ void drawing_classifier::init_options(
       FLEX_UNDEFINED,
       std::numeric_limits<int>::min(), std::numeric_limits<int>::max());
 
+  options.create_boolean_option(
+      "warm_start",
+      "Enable warm start model initialization for training.",
+      false,
+      true);
```

Comment:
What we save in the model state should probably reflect what the user-visible API is: https://github.com/apple/turicreate/blob/2ebb43b1dbe10b369f57ad22c3432eaf62148d28/src/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py#L97

My guess is that this should be a string value, either "quickdraw_245_v0" or the empty string as a proxy for `None`.

(Someday we might be able to handle the model downloading in C++ and sidestep the use of the internal "mlmodel_path" option.)

Reply:
For simplicity I used a boolean because in the API we have:
```
"auto": Uses quickdraw_245_v0
"quickdraw_245_v0": Uses a model trained on 245 of the 345 classes in the
                         Quick, Draw! dataset.
None: No Warm Start"
```
It appears that auto and quickdraw_245_v0 both point to the same warmstart model. So it is simply an option to use a warmstart model or not, hence I captured that as True or False.

Reply:
Since this option is encoded into the state, we should make this more future proof. Also, if the user specifies “quickdraw_245_v0” explicitly during model creation, that value should appear in the model state too

Reply:
@nickjong After the change now, we replace 'auto' with whichever model version is currently our default, i.e. 

- if user passes 'auto' or 'quickdraw_245_v0' in `warm_start`, opts `warm_start` stores “quickdraw_245_v0”
- if user passes 'quickdraw_245_v[X]' in `warm_start`, opts `warm_start` stores “quickdraw_245_v[X]” as we add/support more versions in the future
- if user passes None in `warm_start`, opts `warm_start` stores "None"